contractsV5/IKyberHint.sol

```diff
@@ -19,7 +19,7 @@ interface IKyberHint {
 
     function parseTokenToTokenHint(bytes calldata hint) external view
     returns(HintType tokenToEthType, address[] memory tokenToEthReserves, uint[] memory tokenToEthSplits,
-        HintType ethToTokenType, address[] memory ethToTokenReserves, uint[] memory ethToTokenSplits);
+        HintType ethToTokenType, address[] memory ethToTokenReserves, uint[] memory ethToTokenSplits, uint failureHint);
```

Comment:
I rebased my branch, but it's still showing this change for some reason.

Reply:
yes
I want it indented.
what do u expect?